src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -78,15 +84,26 @@ using namespace ::chip::DeviceLayer::Internal;
 
 void sl_ble_init()
 {
-    uint8_t randomAddrBLE[6] = { 0 };
+    uint8_t randomAddrBLE[RSI_BLE_ADDR_LENGTH] = { 0 };
+#if SIWX_917
+    sl_status_t sl_status;
+    //! Get Random number of desired length
+    sl_status = sl_si91x_trng_get_random_num((uint32_t *)randomAddrBLE, RSI_BLE_ADDR_LENGTH);
```

Comment:
this work is already done in another PR isn't it?  It is a bit confusing to have the same additions in 2 pr

Reply:
removed

Reply:
rebased the branch